Documentation Housekeeping (#2768)
edgedb/edgedb: docs/edgeql/ddl/links.rst

    supertypes.

:eql:synopsis:`SET SINGLE`
    Change the link set's maximum cardinality to *one*.  Only
    valid for concrete links.

:eql:synopsis:`SET MULTI`

>     Change the link set's maximum cardinality to fit any natural number 
>     ( *greater than or equal to one* ). Only valid for concrete links.

Reviewer: ```suggestion
    Remove the upper limit on the link set's cardinality. Only valid for 
    concrete links.
```

I think this wording is clearer, instead of trying to talk about "greater than one" or "natural numbers".

:eql:synopsis:`RESET CARDINALITY`

>     Reset the link set's maximum cardinality to the default value
>     (``SINGLE``), or to the link value inherited from its supertypes.

Reviewer: ```suggestion
    Reset the link set's maximum cardinality to the default value
    (``SINGLE``), or to the value inherited from the link's supertypes.
```

    if there is no implicit or assignment cast from old to new type.

:eql:synopsis:`RESET TYPE`

>     For a given type ``T`` that owns a set of links ``L``, with 
>     ``L = { link_1, link_2, ... , link_t }``, ``RESET TYPE`` morphs the type signature on
>     ``L``'s elements to become the kind allocated on ``T``'s supertype (provided that they
>     share the same name).
>     It is an error to ``RESET TYPE`` on a link that is not inherited.

Reviewer: ```suggestion
    Reset the type of the link to be strictly the inherited type. This only 
    has an effect on links that have been :ref:`overloaded
    <ref_eql_sdl_links_overloading>` in order to change their inherited 
    type. It is an error to ``RESET TYPE`` on a link that is not inherited.
```

I think this formulation addresses what the command does much better.

Examples
--------

> On the object type ``User``, set the ``title`` annotation of its ``friends`` link to
> ``"Friends"``:

Reviewer: ```suggestion
On the object type ``User``, set the ``title`` annotation of its
``friends`` link to ``"Friends"``:
```
